Lib/fontTools/help/__main__.py

```diff
@@ -0,0 +1,35 @@
+"""Show this help"""
+import pkgutil
+import sys
+from setuptools import find_packages
+from pkgutil import iter_modules
+import fontTools
+import importlib
+
+
+def get_description(pkg):
+  try:
+    return __import__(pkg+".__main__",globals(),locals(),["__doc__"]).__doc__
```

Comment:
can you run this through [black](https://github.com/psf/black) so we get consistent formatting? I like to do this at least for newly added modules